website/moneybirdsynchronization/services.py

```diff
@@ -0,0 +1,67 @@
+from payments.models import Payment
+import datetime
+
+def get_financial_account_id(api, name):
```

Comment:
I propose to rewrite this to a class based MoneybirdAPIService that takes `api` at construction

Reply:
Or even better, calls `HttpsAdministration(settings.MONEYBIRD_API_KEY, settings.MONEYBIRD_ADMINISTRATION_ID)` at construction itself